stockflux-securities-master/src/components/TextField.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./TextField.css";
+
+const TextField = props => {
```

Comment:
I think it would be better here to specify the arguments you want to receive in the parameters to the component and use the spread operator to grab the rest of the props:

    const TextField = ({ readOnly, className, ...props }) =>

Reply:
yep added these back in